CustomDynamicFont/Sources/Domain/Entities/Lifecycle.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  Lifecycle.swift
+//  CustomDynamicFont
+//
+//  Created by Bliss on 15/11/21.
+//  Copyright © 2021 Nimble. All rights reserved.
+//
+
+enum Lifecycle {
+
+    case uikit
+    case swiftui
```

Comment:
I think we can update the naming for these enum case to `camelCase` instead as per the [Swift Official's Guidelines](https://www.swift.org/documentation/api-design-guidelines/) 🙏

![Screen Shot 2021-11-16 at 17 27 15](https://user-images.githubusercontent.com/70877098/141968825-8db61bd2-97d7-4e39-bbc8-36a2bdc6348c.png)

```suggestion
    case uiKit
    case swiftUI
```